plugin_blacklist:
- '*_position'
